Microsoft.Toolkit.Uwp.UI.Controls/Loading/Loading.Properties.cs

```diff
@@ -0,0 +1,50 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Loading control allows to show an loading animation with some xaml in it.
+    /// </summary>
+    public partial class Loading
+    {
+        /// <summary>
+        /// EventHandler who enables manipulation of the content before control appears.
+        /// </summary>
+        public event EventHandler LoadingRequired;
```

Comment:
The point of this event isn't clear to me. What does this accomplish that cannot be accomplished right before setting `IsLoading` to true?